test/e2e/testkit/mocks/log_spammer.go

```diff
@@ -0,0 +1,32 @@
+package mocks
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type LogSpammer struct {
+	name      string
+	namespace string
+}
+
+func NewLogSpammer(name, namespace string) *LogSpammer {
+	return &LogSpammer{
+		name:      name + "-spammer",
+		namespace: namespace,
+	}
+}
+
+func (ls *LogSpammer) K8sObject() *corev1.Pod {
+	return &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      ls.name,
+			Namespace: ls.namespace,
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{Name: "flog", Image: "mingrammer/flog", Args: []string{"-b=100", "-f=json", "-l"}},
```

Comment:
Maybe use a simple shell script that emits some output in a loop as a log generator. I remember that this one produces a huge amount of output, which causes unnecessary high load on the k3d cluster and blows up the output file.

Reply:
changed to simple shell commands